packages/react/src/components/PageHeader/PageHeader.tsx

```diff
@@ -17,8 +18,11 @@ import { usePrefix } from '../../internal/usePrefix';
 import { breakpoints } from '@carbon/layout';
 import { useMatchMedia } from '../../internal/useMatchMedia';
 import { Text } from '../Text';
+import { MenuButton } from '../MenuButton';
+import { MenuItem } from '../Menu';
 import { DefinitionTooltip } from '../Tooltip';
 import { AspectRatio } from '../AspectRatio';
+import { createOverflowHandler } from '@carbon/utilities/es/overflowHandler';
```

Comment:
Is it safe to import this from the `es` directory? Or should it be the following instead:

```suggestion
import { createOverflowHandler } from '@carbon/utilities';
```

I will note that when it's imported from the `es` directory, the types for the function are not picked up when it's used.

Reply:
Ah I'm used to importing from the `es` folder when working on the web components package and did the same here - yep I'll adjust that